djangocms_page_sitemap/sitemap.py

```diff
@@ -80,3 +80,20 @@ def changefreq(self, title):
                 return title.page.pagesitemapproperties.changefreq
             except PageSitemapProperties.DoesNotExist:
                 return self.default_changefreq
+
+    def lastmod(self, page_url):
+        # if versionining is enabledcode below is Added to return the latest version modified
```

Comment:
Change to:

        # if versioning is enabled we return the latest version modified using the versioning modified date
        # f versioning is disabled we return the page changed_date